### PR TITLE
Feature/updating to 0.9.0

### PR DESCRIPTION
**Pull request by kiuby88**
Updating brooklyn to 0.9.0 SNAPSHOT version

### CODE REVIEW

File: byon/files/seaclouds-catalog.bom [hunk 1]
```diff
@@ -186,13 +186,13 @@ brooklyn.catalog:
...
       name: Deployer
-      version: 0.8.0-incubating
+      version: 0.9.0-SNAPSHOT
 
       brooklyn.config:
         onExistingProperties: do_not_use
         brooklynnode.webconsole.nosecurity: true
         brooklynnode.classpath:
-          - https://s3-eu-west-1.amazonaws.com/seaclouds-deployer/deployer-0.8.0-20151007.193920-32.jar
+          - https://s3-eu-west-1.amazonaws.com/seaclouds-deployer/deployer-0.8.0-20150812.15.43.24-32-b090.jar
```
Comment (andreaturli) [inline byon/files/seaclouds-catalog.bom]: shouldn't be use 0.9.0 instead?

Comment (andreaturli) [inline byon/files/seaclouds-catalog.bom]: if it is just a naming problem I can merge it as-is.

Comment (kiuby88) [inline byon/files/seaclouds-catalog.bom]: Sorry for the late response, @andreaturli 
I have updated a new jar with a meaningful name

**Comment by kiuby88** [comment]
Thx

**Pull request merged**